src/qml/controls/ValueInput.qml

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+TextEdit {
+    id: root
+    property string description
```

Comment:
Default value?

Reply:
should this be required instead?

Reply:
required if all text input fields should have a description, default value if that's not the case

Reply:
I'm fine with either.

Reply:
giving a default value for now